locale/foreman_theme_satellite.pot

```diff
@@ -134,252 +153,15 @@ msgstr ""
 msgid "(build: %{version})"
 msgstr ""
 
-#: action_names.rb:2
```

Comment:
I wonder why these were listed here.
It's a file from Katello (https://github.com/Katello/katello/blob/master/locale/action_names.rb) and I would expect it to be translated there, not here.

Reply:
foreman-tasks doesn't flag where actions come from. If it's in your activated gems, it gets loaded and put here. We can actually add the file to `.gitignore` and I've started doing so in some plugins (in the translations batch I'm about to push).

Reply:
ah, interesting, yeah I don't have tasks loaded rn. but probably Brian did when originally generating this file in 2018.